cmd/flow-aggregator/config.go

```diff
@@ -21,12 +21,19 @@ type FlowAggregatorConfig struct {
 	// If no L4 transport proto is given, we consider tcp as default.
 	// Defaults to "".
 	ExternalFlowCollectorAddr string `yaml:"externalFlowCollectorAddr,omitempty"`
-	// Provide flow export interval as a duration string. This determines how often the flow aggregator exports flow
-	// records to the flow collector.
-	// Flow export interval should be greater than or equal to 1s (one second).
-	// Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".
-	// Defaults to "60s".
-	FlowExportInterval string `yaml:"flowExportInterval,omitempty"`
+	// Provide the active flow record timeout as a duration string. This determines
+	// how often the flow aggregator exports the active flow records to the flow
+	// collector. Thus, for flows with a continuous stream of packets, a flow record
+	// will be exported to the collector once the elapsed time since the last export
+	// event is equal to the value of this timeout.
+	// Defaults to "60s". Follow the time units of duration type.
```

Comment:
same comment as above: why did you switch to `Follow the time units of duration type.`?